src/schemapack/_internals/validation/plugins/missing_origin.py

```diff
@@ -87,9 +87,7 @@ def validate(
             raise ValidationPluginError(
                 type_="MissingMandatoryOriginError",
                 message=(
-                    "For the following relations that are mandatory on the origin end"
-                    + " the specified targets are not referenced by any origin:"
-                    + " {not_referenced_target_by_relation}"
+                    f"For the following relations that are mandatory on the origin end the specified targets are not referenced by any origin: '{not_referenced_target_id_by_relation}'"
```

Comment:
Shall we keep the line length / string split as it was before and just make the last segment an f-string?